pkg/scheduler/framework/session.go

```diff
@@ -53,6 +56,10 @@ import (
 	"volcano.sh/volcano/pkg/scheduler/util"
 )
 
+const (
+	ClusterTopHyperNode = "<cluster-top-hypernode>"
```

Comment:
Could you add some comments to explain what is this `ClusterTopHyperNode` const for?